runelite-api/src/main/java/net/runelite/api/ItemID.java

```diff
@@ -3421,11 +3421,11 @@
 	public static final int BOOK_OF_FOLKLORE = 5508;
 	public static final int SMALL_POUCH = 5509;
 	public static final int MEDIUM_POUCH = 5510;
-	public static final int MEDIUM_POUCH_5511 = 5511;
+	public static final int MEDIUM_POUCH_DAMAGED = 5511;
```

Comment:
These names are automatically generated so you can't really rename them. Instead make a ```public static final int MEDIUM_POUCH_DAMAGED = MEDIUM_POUCH_5511``` in the runecraft overlay maybe?